sdks/python/apache_beam/runners/worker/worker_status.py

```diff
@@ -230,7 +230,7 @@ def close(self):
   def _log_lull_in_bundle_processor(self, bundle_process_cache):
     while True:
       time.sleep(2 * 60)
-      if bundle_process_cache.active_bundle_processors:
+      if not bundle_process_cache and bundle_process_cache.active_bundle_processors:
```

Comment:
misplaced `not` ?

Reply:
Oh, I inverted that one. Thanks for catching!